www/_data/fetched-files.yml

```diff
@@ -29,41 +29,13 @@
         commit: "master"
     dest:
         path: "reference/cordova-plugin-camera/index.md"
--
-    src:
-        repoName: "apache/cordova-plugin-console"
-        packageName: "cordova-plugin-console"
-        commit: "master"
-    dest:
-        path: "reference/cordova-plugin-console/index.md"
```

Comment:
Should this one have gotten removed?

Reply:
Yes, I think so as https://github.com/apache/cordova-plugin-console mentions it is deprecated and included in the platforms directly.